test/e2e/placement_eviction_test.go

```diff
@@ -11,18 +11,309 @@ import (
+		}
+	})
+})
+
+var _ = Describe("ClusterResourcePlacement eviction of bound binding, no taint specified, evicted cluster is picked again by scheduler - No PDB specified", Ordered, Serial, func() {
+	crpName := fmt.Sprintf(crpNameTemplate, GinkgoParallelProcess())
+	crpEvictionName := fmt.Sprintf(crpEvictionNameTemplate, GinkgoParallelProcess())
+
+	BeforeAll(func() {
+		By("creating work resources")
+		createWorkResources()
+
+		// Create the CRP.
+		createCRP(crpName)
+	})
+
+	AfterAll(func() {
+		ensureCRPEvictionDeleted(crpEvictionName)
+		ensureCRPAndRelatedResourcesDeleted(crpName, allMemberClusters)
+	})
+
+	It("should update cluster resource placement status as expected", func() {
+		crpStatusUpdatedActual := crpStatusUpdatedActual(workResourceIdentifiers(), allMemberClusterNames, nil, "0")
+		Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement status as expected")
+	})
+
+	It("should place resources on the all available member clusters", checkIfPlacedWorkResourcesOnAllMemberClusters)
+
+	It("create cluster resource placement eviction targeting member cluster 1", func() {
+		crpe := &placementv1alpha1.ClusterResourcePlacementEviction{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: crpEvictionName,
+			},
+			Spec: placementv1alpha1.PlacementEvictionSpec{
+				PlacementName: crpName,
+				ClusterName:   memberCluster1EastProdName,
+			},
+		}
+		Expect(hubClient.Create(ctx, crpe)).To(Succeed(), "Failed to create CRP eviction %s", crpe.Name)
+	})
+
+	It("should update cluster resource placement eviction status as expected", func() {
+		crpEvictionStatusUpdatedActual := testutilseviction.StatusUpdatedActual(
+			ctx, hubClient, crpEvictionName,
+			&testutilseviction.IsValidEviction{IsValid: true, Msg: condition.EvictionValidMessage},
+			&testutilseviction.IsExecutedEviction{IsExecuted: true, Msg: condition.EvictionAllowedNoPDBMessage})
+		Eventually(crpEvictionStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement eviction status as expected")
+	})
+
+	It("should ensure evicted cluster is picked again by scheduler & update cluster resource placement status as expected", func() {
+		crpStatusUpdatedActual := crpStatusUpdatedActual(workResourceIdentifiers(), allMemberClusterNames, nil, "0")
+		Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement status as expected")
+	})
+
+	It("should still place resources on the all available member clusters", checkIfPlacedWorkResourcesOnAllMemberClusters)
+})
+
+var _ = Describe("ClusterResourcePlacement eviction of bound binding - PickAll CRP, PDB specified to protect resources on all clusters, eviction denied", Ordered, func() {
```

Comment:
is there a pickFixed case? Is there a pickAll but PDB specified maxUnavailabe case?